README.md

```diff
@@ -1,7 +1,186 @@
-# kontent-html-smart-link-decorator-js
+# kontent-smart-link
 
-  Html element decorator allowing to inject [smart links](https://docs.kontent.ai/tutorials/develop-apps/build-strong-foundation/set-up-editing-from-preview#a-using-smart-links) to Kentico Kontent according to specified [HTML data attributes](https://www.w3schools.com/tags/att_data-.asp).
+![licence](https://img.shields.io/github/license/Kentico/kontent-smart-link)
+![npm](https://img.shields.io/npm/v/@kentico/kontent-smart-link)
+![downloads](https://img.shields.io/npm/dt/kontent-smart-link)
+![jsdelivr](https://img.shields.io/jsdelivr/npm/hm/@kentico/kontent-smart-link)
+![snyk](https://img.shields.io/snyk/vulnerabilities/github/Kentico/kontent-smart-link)
+
+Html element decorator allowing to inject [smart links](https://docs.kontent.ai/tutorials/develop-apps/build-strong-foundation/set-up-editing-from-preview#a-using-smart-links) to Kentico Kontent according to specified [HTML data attributes](https://www.w3schools.com/tags/att_data-.asp).
 
 ## ⚠ Disclaimer
 
-This project/repository in a work in progress and is not ready for production use yet. The features and APIs might be (and probably will be) changed.
+This project/repository is a work in progress and is not ready for production use yet. The features and APIs might be (and probably will be) changed.
```

Comment:
This should be removed now, shouldn't it?

Reply:
I don't think so. This is only the ETP version of the package (0.0.1) and we still may change some APIs of the SDK (e.g. while adding support for components). This message will be removed when we will release the 1.0.0 version of the package.